src/components/common/TicketEditor/TicketEditor.tsx

```diff
@@ -119,14 +119,20 @@ const TicketEditor = ({ ticketData }: TicketEditorProps) => {
 
   const handleTicketBuilder = async () => {
     try {
-      const ticketForReview = {
-        ...ticketData,
-        name,
-        description,
-        status: 'DRAFT' as TicketStatus
+      const ticketPayload = {
+        metadata: {
+          source: 'websocket',
+          id: ticketData.uuid
```

Comment:
@MuhammadUmer44 - This has the same mistake as https://github.com/stakwork/sphinx-tribes-frontend/pull/709

The metadata.id is the websocket session ID. not the ticketuuid which is already present in ticket.uuid.

Websocket is opened nad logged on page load of PhasePlanner page

Reply:
@humansinstitute, Fixed